Sources/CoreMetrics/Metrics.swift

```diff
@@ -347,8 +362,8 @@ public extension Timer {
 /// * `Recorder` -> `RecorderHandler`
 /// * `Timer` -> `TimerHandler`
 ///
-/// This type is an implementation detail and should not be used directly, unless implementing your own metrics backend.
-/// To use the SwiftMetrics API, please refer to the documentation of `MetricsSystem`.
+/// - warning: This type is an implementation detail and should not be used directly, unless implementing your own metrics backend.
```

Comment:
I wonder if also all "We do not expect this API to be used in normal circumstances, so if you find yourself using it make sure it's for a good reason." should be made a warning...?

Reply:
god idea +1

Reply:
ok will do 👍